lib/VM/StorageProvider.cpp

```diff
@@ -63,7 +63,7 @@ llvh::ErrorOr<void *> VMAllocateStorageProvider::newStorageImpl(
   }
   void *mem = *result;
   assert(isAligned(mem));
-  (void)isAligned;
+  (void)mem;
```

Comment:
This is technically not a spelling mistake. There is a compiler warning about it. I just considered it as a typo. I assume that the intent was to avoid a warning about unused `mem` variable in Release mode and when the `HERMESVM_ALLOW_HUGE_PAGES` is not defined.

Reply:
It turned out the the problem here was `isAligned()` being unused. Can you please revert it?

Reply:
Can we have an alternative solution for the `isAligned()`? The code `(void)isAligned()` produces the following warning in MSVC:
```
lib\VM\StorageProvider.cpp(66,9): warning C4551: function call missing argument list [build\lib\VM\hermesVMRuntimeLean.vcxproj]
```
Is there a warning in Clang?

Reply:
Reverted. I will submit a fix for the warning as a separate PR.